rss/serializers/atom_1_0.py

```diff
@@ -1,15 +1,81 @@
 """Serializer for Atom 1.0"""
 
-from flask.json import jsonify
+from feedgen.feed import FeedGenerator
```

Comment:
This may seem minor (and it is), but if I recall correctly, the linting settings we are using likes our imports to be alphabetized, both going down the "from" list, and within the "import" list.

Reply:
I followed the pylint complaints about reordering the imports until they were all gone.  That didn't seem to require changing the order of the imports within a "from".  I also changed all of the imports to be of the form "from X import Y", and hope that is OK.